<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Face Rigger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All the locators and the curves, must be inside a group called,&lt;/p&gt;&lt;p&gt;base_head_rig_grp, locators names must end in _pnt and&lt;/p&gt;&lt;p&gt;the curves in _crv&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="impRig_btn">
      <property name="text">
       <string>1.Import Face Rig Template</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Curves must have 2 attributes, controls_count and use_tips</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addAttrToCurve_btn">
        <property name="text">
         <string>Add Attributes to Curve</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mesh To Deform:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="meshToDeform_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="fillMeshToDeform_btn">
        <property name="text">
         <string>2.&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Verteces To Ignore:</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fillVertecesToIgnore_btn">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="vertecesToIgnore_lw"/>
    </item>
    <item>
     <widget class="QPushButton" name="createSofts_btn">
      <property name="text">
       <string>3.Create SoftMods</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Mirror Softs Settings</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="mirrorSoftRtoL_btn">
        <property name="text">
         <string>Right to Left</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mirrorSoftLtoR_btn">
        <property name="text">
         <string>Left To Right</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Mesh To Skin:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="meshToSkin_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="fillMeshToSkin_btn">
        <property name="text">
         <string>4.&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adjust Soft Mods, settings... FallOff Mode and FallOff Radius, &lt;/p&gt;&lt;p&gt;and then click &amp;quot;Create Face Rig&amp;quot; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Skin:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="skin_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="fillSkin_btn">
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="create_btn">
      <property name="text">
       <string>5.Create Face Rig</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
